C/Compose/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Compose"
+version = v"2.17.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://compose.obspm.fr/download/code/codehdf5.zip",
+                  "f3f68203a50bb898abe31ee0b3dc750a1f1164c9e5d7fb9c4546b4eaa0cd172b"),
+    FileSource("https://compose.obspm.fr/home", "d5aa9cb98faf17b6d9d98544ced475922c138ce79bb5282f995496e9927ec4b1";
+               filename="homepage.html"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ ${target} == x86_64-linux-musl ]]; then
+    # HDF5 needs libcurl, and it needs to be the BinaryBuilder libcurl, not the system libcurl.
+    rm /usr/lib/libcurl.*
+    rm /usr/lib/libnghttp2.*
+fi
+
+cd $WORKSPACE/srcdir
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/output.patch
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/errors.patch
+
+# Make `h5fc` available
+mkdir bin
+ln -s $bindir/h5pfc bin/h5fc
+export PATH=$PATH:$WORKSPACE/srcdir/bin
+
+make compose test_read_hdf5 test_read_opacity
+
+install -Dm 755 "compose${exeext}" "${bindir}/compose${exeext}"
+install -Dm 755 "test_read_hdf5${exeext}" "${bindir}/test_read_hdf5${exeext}"
+install -Dm 755 "test_read_opacity${exeext}" "${bindir}/test_read_opacity${exeext}"
```

Comment:
I prefer more verbose output in the log for future reference
```suggestion
install -Dvm 755 "compose${exeext}" "${bindir}/compose${exeext}"
install -Dvm 755 "test_read_hdf5${exeext}" "${bindir}/test_read_hdf5${exeext}"
install -Dvm 755 "test_read_opacity${exeext}" "${bindir}/test_read_opacity${exeext}"
```